webapp/app/js/config/sample.local.conf.js

```diff
@@ -3,17 +3,19 @@
  * file, you can obtain one at http://mozilla.org/MPL/2.0/. */
 
 'use strict';
+var production = "https://treeherder.mozilla.org";
+var stage = "https://treeherder.allizom.org";
+var dev = "http://treeherder-dev.allizom.org";
+var same_host = "";
```

Comment:
It is always the same as something else - the same as the UI host name :-)
(See block comment below)
Though perhaps this needs to be re-ordered to be clearer.

Or we could just call it 'vagrant' and ignore prod/stage for now, since they just don't have local.conf.js at all (even though they use the empty string effectively too - which is why I'd made the variable more generic and called it same_host)